docker-compose.override.dev.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.2"
+
+services:
+  # service configuration for our dockerized Rails app
+
+  netrunner_cards_json:
+    build: https://github.com/NetrunnerDB/netrunner-cards-json.git#main
+
+  nrdb_api_server:
+    image: registry.digitalocean.com/nsgcr/nrdb_api_server:dev
+    # build the Dockerfile next to this file, tag the built image with above image name
+    build: .
+    command:
+      - /bin/sh
+      - -c
+      - |
+        rake db:reset
+        bundle exec rails cards:import
```

Comment:
To be discussed, this resets the db and re-imports cards everytime, which could be fine for dev workflows, it just takes a few more seconds to wait for everything to be ready after a start, or restart of the containers